evadb/catalog/catalog_utils.py

```diff
@@ -43,6 +43,8 @@
 def generate_sqlalchemy_conn_str(engine: str, params: Dict[str, str]):
     if engine == "postgres":
         conn_str = f"""postgresql://{params["user"]}:{params["password"]}@{params["host"]}:{params["port"]}/{params["database"]}"""
+    elif engine == "mysql":
+        conn_str = f"""mysql://{params["user"]}:{params["password"]}@{params["host"]}:{params["port"]}/{params["database"]}"""
```

Comment:
This is not needed. @jiashenC Shall we remove / cleanup this function to avoid confusions?

Reply:
Yes, we should remove that